runtime/astar/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "astar-runtime"
-version = "5.1.0"
+version = "5.1.1"
```

Comment:
You should bump up the minor version since this is a new, backwards compatible functionality :)